dkpro-core-opennlp-asl/src/scripts/build.xml

```diff
@@ -45,32 +45,32 @@
        <mkdir dir="target/download"/>
 
        <get 
-             src="https://bintray.com/inofficalbinarystore/generic/download_file?file_path=opennlp-af-afribooms.tar.gz" 
-             dest="target/download/opennlp-af-afribooms.gz" 
+             src="https://bintray.com/inofficalbinarystore/generic/download_file?file_path=opennlp-af-afribooms-perceptron.tar.gz" 
+             dest="target/download/opennlp-af-afribooms-perceptron.gz" 
              skipexisting="true"/>
        <untar 
-             src="target/download/opennlp-af-afribooms.gz" 
+             src="target/download/opennlp-af-afribooms-perceptron.gz" 
              dest="target/download/" 
              compression="gzip"/>
   
        <install-stub-and-upstream-file 
-          file="target/download/opennlp-af-afribooms/model-tagger-af.bin" 
+          file="target/download/opennlp-af-afribooms-perceptron/model-tagger-af.bin" 
           groupId="de.tudarmstadt.ukp.dkpro.core"
-          md5="cb9be0243fe7908d00a0749b760b81f5"
+          md5="491e782487531a9501e77e6b522d57d8"
           artifactIdBase="de.tudarmstadt.ukp.dkpro.core.opennlp"
-          upstreamVersion="20181017"
+          upstreamVersion="20181021"
           metaDataVersion="1"
           tool="tagger"
           language="af"
-          variant="afribooms"
+          variant="afribooms-perceptron"
           extension="bin">
             <metadata>
                <entry key="DC.rights" value="https://creativecommons.org/licenses/by-sa/4.0/"/>
                <entry key="pos.tagset" value="ud"/>
                <entry key="model.encoding" value="utf-8"/>
             </metadata>
             <additionalUpstreamContent>
-               <copy file="target/download/opennlp-af-afribooms/ref.cff" todir="target/model-staging/META-INF/"/>
+               <copy file="target/download/opennlp-af-afribooms-perceptron/ref.cff" todir="target/model-staging/META-INF/"/>

```

Comment:
I would suggest to also include a copy of the license text as a file.

Reply:
Are you still planning to add more metadata such as tag statistics, genre, etc?

Reply:
No, I will limit it to the current set of meta data.